x11-terms/guake/guake-3.9.0.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..11} )
+inherit distutils-r1 gnome2-utils plocale virtualx xdg
+
+PLOCALES="ca cs de el es fa fi fr gl hr hu id it ja ko nb nl pa pl pt_BR ru sv tr uk zh_CN zh_TW"
+
+DESCRIPTION="Drop-down terminal for GNOME"
+HOMEPAGE="http://guake-project.org/"
+SRC_URI="https://github.com/Guake/guake/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+IUSE="utempter"
```

Comment:
This use flag has no function in this ebuild, we need to move it under `optfeature` instead.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

I can do it.